api/src/controllers/changes.js

```diff
@@ -391,6 +391,7 @@ var getReplicationKey = function(doc) {
   if (doc._id === 'resources' ||
```

Comment:
the WARNING comment about this points to the wrong place now

Reply:
Fixed.